src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -270,29 +270,50 @@ extern void setDbPrefix(std::string _dbPrefix)
 *
 * getOrionDatabases -
 */
-extern void getOrionDatabases(std::vector<std::string>& dbs)
+extern bool getOrionDatabases(std::vector<std::string>& dbs)
```

Comment:
The function is called from contextBrokerInit(), mongoInit() and subscriptionCache::fillFromDb(). None of that places check the return value, but anyway I think that from a logical point of view it is better is getOrionDatabase() signales sucess/failure on DB operation. Currently we are not using that information, but we may use in the future (e.g. to log error at a higher level).